nodeup/pkg/model/kubelet.go

```diff
@@ -186,6 +186,8 @@ func (b *KubeletBuilder) buildKubeconfig() (string, error) {
 		return "", fmt.Errorf("error encoding CA certificate: %v", err)
 	}
 
+	cluster.Server = b.Cluster.Spec.MasterInternalName
```

Comment:
The master really should use 127.0.0.1:8080.  It means we don't rely on DNS being available, which speeds up bringup.  We do have `b.IsMaster` available...